apps/storybook/.storybook/main.ts

```diff
@@ -4,7 +4,14 @@ import type { StorybookConfig } from '@storybook/react-vite';
 const config: StorybookConfig = {
   async viteFinal(config, { configType }) {
     return mergeConfig(config, {
-      define: { 'process.env': {} },
+      define: {
+        'process.env': {},
+      },
+      resolve: {
+        alias: {
+          '@evans': '../../next-evans/components',
```

Comment:
this (and the corresponding change in `tsconfig.json`) lets us have prettier imports inside storybook